aws_lambda_builders/workflows/java_gradle/actions.py

```diff
@@ -72,7 +76,13 @@ def execute(self):
     def _copy_artifacts(self):
         lambda_build_output = os.path.join(self.build_dir, 'build', 'distributions', 'lambda-build')
         try:
-            for f in self.os_utils.listdir(lambda_build_output):
-                self.os_utils.copy(os.path.join(lambda_build_output, f), self.artifacts_dir)
+            self.os_utils.makedirs(self.artifacts_dir)
```

Comment:
os.scandir is not available in Python 2.7. Can you use this osutils helper instead?

https://github.com/awslabs/aws-lambda-builders/blob/develop/aws_lambda_builders/workflows/python_pip/utils.py#L56-L66

Copy/paste it in your code for now.